qiskit/pulse/qasm_def.py

```diff
@@ -0,0 +1,203 @@
+# -*- coding: utf-8 -*-
```

Comment:
I think this name should reflect closer to the class name since it is a module for only a single class.

Reply:
yes you are right 👍